randovania/games/dread/json_data/Dairon.txt

```diff
@@ -1417,6 +1417,8 @@ Extra - asset_id: collision_camera_018
 > Dock to EMMI Zone Exit North; Heals? False
   * Layers: default
   * Open Passage to EMMI Zone Exit North/Dock to Yellow EMMI Introduction
+  > Dock to Big Hub
+      Speed Booster and After Dairon - Yellow Introduction Speed Blocks Destroyed
```

Comment:
This connection is redundant. It's already handled from "Door to EMMI Zone Exit East"

Reply:
I missed that, the point was to do it trickless. Given that I found other cases that indicate that Speed Booster can be used tricklessly to pass Shutter Gates, I changed the existing connection by moving the Speed Booster branch outside the Flash Shift Skip branch.